features/test/src/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Test",
+    "author": "polypoly Cooperative",
+    "description": "Semi-automatic testing for the polyPod APIs - please ignore",
+    "thumbnail": "images/thumbnail.pdf",
+    "primaryColor": "#cbd5e0",
+    "borderColor": "#cbd5e0",
```

Comment:
Not really important, but maybe slightly change these colors so that they are not the same"

Reply:
Not important enough, anyone can do that when they want to.